backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/DataTableUtil.java

```diff
@@ -359,4 +359,15 @@ public String extractValueOrNull(@Nullable final String value)
 		return value;
 	}
 
+	@Nullable
+	public String extractNullableStringForColumnName(@NonNull final Map<String, String> dataTableRow, @NonNull final String columnName)
+	{
+		return dataTableRow.get(columnName);
+	}
+
+	@Nullable
+	public String nullToken2Null(@NonNull final String value)
+	{
+		return NULL_STRING.equals(value) ? null : value;
```

Comment:
```suggestion
		return NULL_STRING.equals(value.trim()) ? null : value;
```

Reply:
@metas-ts  careful with that axe, Eugen!

if value is null u might get NPE.

Reply:
@teosarca i'm pretty sure that this point it's going to be `@NonNull` 🤓

Reply:
ahh fuck, no i saw the `@NonNull`! nvm me